<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Меню</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/database.png</normaloff>../icons/database.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>841</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/add.png</normaloff>../icons/add.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/search.png</normaloff>../icons/search.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить запись</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/cancel.png</normaloff>../icons/cancel.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>841</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>0</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/refresh.png</normaloff>../icons/refresh.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Параметры</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Выбор базы данных</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionSQLite"/>
     <addaction name="actionMySQL_MariaDB"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSQLite">
   <property name="text">
    <string>SQLite</string>
   </property>
  </action>
  <action name="actionMySQL_MariaDB">
   <property name="text">
    <string>MariaDB</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
